src/Android/Xamarin.Android/Samples/Data/AddGeoPackageFeatureTable/AddGeoPackageFeatureTable.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using System.Linq;
+using System.IO;
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+using ArcGISRuntimeXamarin.Managers;
+using System.Threading.Tasks;
+using Esri.ArcGISRuntime.Data;
+
+namespace ArcGISRuntimeXamarin.Samples.AddGeoPackageFeatureTable
+{
+    [Activity]
+    public class AddGeoPackageFeatureTable : Activity
+    {
+        private MapView _myMapView;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Add GeoPackage feature table";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+        
+        private async void Initialize()
+        {
+            // Create a new map centered on Aurora Colorado
+            _myMapView.Map = new Map(BasemapType.LightGrayCanvas, 39.7294, -104.8319, 9);
+
+            // Get the full path
+            string geoPackagePath = await GetGeoPackagePath();
+
+            // Open the GeoPackage
+            GeoPackage myGeoPackage = await GeoPackage.OpenAsync(geoPackagePath);
+
+            // Read the feature tables and get the first one
+            FeatureTable geoPackageTable = myGeoPackage.GeoPackageFeatureTables.FirstOrDefault();
+
+            // Make sure a feature table was found in the package
+            if (geoPackageTable == null) { return; }
+
+            // Create a layer to show the feature table
+            FeatureLayer newLayer = new FeatureLayer(geoPackageTable);
+            await newLayer.LoadAsync();
+
+            // Add the feature table as a layer to the map (with default symbology)
+            _myMapView.Map.OperationalLayers.Add(newLayer);
+        }
+
+        private async Task<string> GetGeoPackagePath()
+        {
+            #region offline data
+
+            // The GeoPackage will be downloaded from ArcGIS Online.
+            // The data manager (a component of the sample viewer), *NOT* the runtime handles the offline data process
+
+            // The desired GPKG is expected to be called "AuroraCO.shp"
```

Comment:
It looks like this comment didn't get changed. If this is the expected behavior then there is no problem, otherwise this will need to change for each implementation.